x-pack/test/alerting_api_integration/security_and_spaces/group2/tests/actions/sub_action_framework/index.ts

```diff
@@ -314,6 +374,179 @@ export default function createActionTests({ getService }: FtrProviderContext) {
           service_message: 'You should register at least one subAction for your connector type',
         });
       });
+
+      describe('SentinelOne Connector sub-actions authz', () => {
```

Comment:
This test seems specific to SentinelOne and not related to the sub-action framework. What do you think about moving the tests in `x-pack/test/alerting_api_integration/security_and_spaces/group2/tests/actions/connector_types` and add a test here to test the framework's authorize by using a test sub action connector? For example `x-pack/test/alerting_api_integration/common/plugins/alerts/server/sub_action_connector.ts`.

Reply:
Sure @cnasikas - I can do that. I will work on it today